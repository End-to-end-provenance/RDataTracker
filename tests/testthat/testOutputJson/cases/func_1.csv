"ddg.pnum","ddg.fun","ddg.fnum","ddg.lib","ddg.lnum"
2,"read.csv",1,"utils",7
