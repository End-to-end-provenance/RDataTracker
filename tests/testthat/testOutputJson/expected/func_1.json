
		// function nodes
		"rdt:f1": {
			"name": "read.csv"
		}
